src/app/Scheduler/Program.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Common.Jobs;
+using Common.Utils;
+using Hangfire;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Scheduler.Jobs;
+using Scheduler.Settings;
+using Scheduler.Settings.JobConfigs;
+using Serilog;
+using Serilog.Events;
+using Serilog.Formatting.Compact;
+
+using var host = createHostBuilder(args).Build();
+
+var hostEnvironment = host.Services.GetRequiredService<IHostEnvironment>();
+
+Log.Logger = buildLogger();
+
+scheduleRecurringJobs();
```

Comment:
An idea for improvement: to add also a sample for ondemand job triggered from API. @yarmolovichalex you can do it later when have time or assign to Alexey.